media/vscode.css

```diff
@@ -0,0 +1,91 @@
+:root {
```

Comment:
Where did you get these styles?
I'd rather not put too much time into visual design at this point since we might get an actual designer for this plugin eventually.